fuzz/fuzz_targets/peer_crypt_target.rs

```diff
@@ -34,13 +31,17 @@ pub fn do_test(data: &[u8]) {
 		Ok(key) => key,
 		Err(_) => return,
 	};
+	let ephemeral_key = match SecretKey::from_slice(get_slice!(32)) {
+		Ok(key) => key,
+		Err(_) => return,
+	};
 
 	let mut crypter = if get_slice!(1)[0] != 0 {
 		let their_pubkey = match PublicKey::from_slice(get_slice!(33)) {
```

Comment:
This will fail a lot.
Every time the first byte won't be 0x02 or 0x03 it will fail.

Reply:
Maybe add a function that generates a Secret Key and creates a Public Key from it.
or hard code 0x02 and randomize the other 32 bytes. (the first one will at least guarantee success)

Reply:
Yea, this test isnt really great, but its also testing very small surface area, so Im not gonna worry *too* much about it.

Reply:
Edit: As you told me before, `get_slice!()` doesn't use randomness.
So maybe using 0x02/0x03 as the first byte and the rest from the provided data should cover most errors.